data-prepper-plugins/kafka-source/README.md

```diff
@@ -0,0 +1,150 @@
+# Kafka source
+
+This is the Data Prepper Kafka source plugin that reads records from Kafka topic. It uses the consumer API provided by Kafka to read messages from the broker.
+
+The Kafka source plugin supports OpenSearch 2.0.0 and greater.
```

Comment:
how is Opensearch version related to kafka source plugin? did we mean to say kafka version?

Reply:
Yes, let's remove this line if it is related to OpenSearch.